articles/openshift/howto-using-aro.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Using `az aro` | Microsoft Docs
+description: Create, access, and delete clusters with `az aro` extension
+author: klamenzo 
+ms.author: b-lejaku
+ms.service: container-service
+ms.topic: conceptual
+ms.date: 03/02/2020
+keywords: aro, openshift, az aro, red hat, cli
+#Customer intent: As a customer, I want to create an ARO custer using the command line.
+---
+
+# Using `az aro`
+
+The `az aro` extension allows you to create, access, and delete Azure Red Hat OpenShift clusters directly from the command line using the Azure CLI.
+
+## Installing the extension
+
+1. Install the [`az`](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) client, if you have not already. You will need `az` version 2.0.72 or greater, as this version includes the `az network vnet subnet update --disable-private-link-service-network-policies` flag.
+
+2. Log in to Azure.
+
+   ```
+   az login
+   ```
+
+3. Run the following command to install the `az aro` extension:
+
+   ```
+   az extension add --source https://arosvc.blob.core.windows.net/az-preview/aro-0.1.0-py2.py3-none-any.whl
+   ```
+
+4. Verify the ARO extension is registered.
+
+   ```
+   az -v
+   ...
+   Extensions:
+   aro                                0.1.0 (dev) /path/to/rp/python/az/aro
+   ...
+   Development extension sources:
+       /path/to/rp/python
+   ...
+   ```
+
+
+## Registering the resource provider
+
+To opt-in for the preview you need to register the `Microsoft.RedHatOpenShift` resource provider.
+
+```
+az provider register -n Microsoft.RedHatOpenShift --wait
+```
+
+
+## Prerequisites to create an Azure Red Hat OpenShift v4 cluster
+
+You need the following items to create an Azure Red Hat OpenShift v4 cluster.
+
+* A cluster AAD application (client ID and secret) and service principal, or sufficient AAD permissions for `az aro` create to create these for you automatically.
+* The resource provider service principal and cluster service principal must each have the Contributor role on the cluster VNet. If you have the User Access Administrator role on the VNet, `az aro` create will set up the role assignments for you automatically.
+* A VNet containing two empty subnets, each with no network security group attached. Your cluster will be deployed into these subnets. Follow the steps below to create your VNet.
+
+### Create a VNet containing two empty subnets
+
+Follow these steps to create a VNet containing two empty subnets.
+
+1. Set the following varibales.
+
+   ```
+   LOCATION=eastus
+   RESOURCEGROUP="v4-$LOCATION"
+   CLUSTER=cluster
+   ```
+
+2. Create a resource group for your cluster.
+
+   ```
+   az group create -g "$RESOURCEGROUP" -l $LOCATION
+   ```
+
+3. Create the VNet containing two empty subnets.
+
+   ```
+   az network vnet create \
+     -g "$RESOURCEGROUP" \
+     -n dev-vnet \
+     --address-prefixes 10.0.0.0/9 \
+     >/dev/null
```

Comment:
we should handle the case where they already have a pre-existing vnet that they want to use, it's a value item of ARO v4 over v3.